packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -416,7 +428,15 @@ const estimateFee = async (request: Request<MessageTypes.EstimateFee>) => {
     const priorityFee = await getPriorityFee(api.rpc, message, transaction.signatures);
     const baseFee = await getBaseFee(api.rpc, message);
     const accountCreationFee = isCreatingAccount
-        ? await api.rpc.getMinimumBalanceForRentExemption(BigInt(getTokenSize())).send()
+        ? await api.rpc
+              .getMinimumBalanceForRentExemption(
+                  BigInt(
+                      creatingTokenAccountType === 'spl-token-2022'
+                          ? getToken2022Size()
+                          : getTokenSize(),
+                  ),
+              )
```

Comment:
nit: perhaps a `_getTokenSize(programName)` helper would make this more readable. Or rather a `getTokenSize` helper and the imported fns could be prefixed with `_`.